### DISCUSSION TITLE
I'm trying to livestream using Logic/Garageband, I am getting a doubling effect that I cannot figure out.

### DISCUSSION THREAD

**Discussion by Jcut4578** [ExistentialAudio/BlackHole#691] [Q&A]
I made a video on exactly what's going on. I've tried pretty much every combination of settings and I just cannot get the sound to go away. Every video or tutorial I've watched or read tells me to set it up like this. 

https://www.youtube.com/watch?v=0U6OB1Y7Dcw&ab_channel=JesseyCutter

Thank you, 
Jessey

**Comment by devinroth** [comment]
What’s happening is you are capturing your microphone twice. The degradation of audio quality is from phasing from the two signals being slightly off from each other.

Don’t capture your mic directly from OBS and BlackHole via Logic at the same time. Use either one or the other.

Devin

**Comment by Jcut4578** [comment]
Yep, that was it. Idk how I didn't try that but thank you so much, working great.

**Comment by devinroth** [comment]
You’re welcome.

I wish more questions were from people like you. You did your due diligence trying to figure it out. And when you got stuck you posted a clear question with steps on what you tried.

So thanks. You made it really easy to help you out.

Other people should take note.

Devin

**Comment by Jcut4578** [comment]
Of course man. I can only imagine how many people just posting easily solvable issues without troubleshooting first. I'm not in tech as you can probably tell but I'm an ICU nurse so troubleshooting is my life. Thanks again for the help as well as this product.
